src/utils.cpp

```diff
@@ -114,4 +114,27 @@ std::vector<std::string> get_kernel_cflags(
   return cflags;
 }
 
+std::string is_deprecated(std::string &str)
+{
+
+  std::vector<DeprecatedName>::iterator item;
+
+  for (item = DEPRECATED_LIST.begin(); item != DEPRECATED_LIST.end(); item++)
```

Comment:
```suggestion
  for (auto item : DEPRECATED_LIST)
```

Reply:
I need it to be pointer so I can do this: `item->show_warning = false`. And stop printing messages for the same value after the first time.